test/tables.mediawiki

```diff
@@ -1,145 +1,152 @@
 Simple table with caption:
 
-{|
+{| class="wikitable"
 |+ Demonstration of simple table syntax.
-!align="right"| Right
-! Left
-!align="center"| Center
+|-
+! style="text-align: right;"| Right
+! style="text-align: left;"| Left
+! style="text-align: center;"| Center
 ! Default
```

Comment:
Can you comment on the change with align?
Is the use of `style: text-align` now recommended for wikipedia tables?

Reply:
Yes, this is recommended. [This edit from 2013](https://www.mediawiki.org/w/index.php?title=Help:Tables&diff=842660&oldid=842627) states "killing deprecated align= attributes", so it has been recommended to use `style: text=align` instead of the `align` attributes since then.

In general, mediawiki markup is very close to html markup, and the discussion on whether `(text-)align` on table elements should be an attribute or style tag for html is not very clear from google, but it seems like `style` tags for align have support since ~2000, and `align` attributes have been deprecated for some time, depending on the source (at least for html5).

I like to follow the [guide in my first comment](https://www.mediawiki.org/w/index.php?title=Help:Tables), which uses style tags since 2013. It seems to be an official source.